library/std/src/ffi/os_str.rs

```diff
@@ -265,6 +266,43 @@ impl OsString {
         self.inner.reserve(additional)
     }
 
+    /// Tries to reserve capacity for at least `additional` more elements to be inserted
+    /// in the given `OsString`. The collection may reserve more space to avoid
```

Comment:
OsString would not normally be conceptualized as a "collection".

```suggestion
    /// in the given `OsString`. The string may reserve more space to avoid
```